Cargo.toml

```diff
@@ -1,18 +1,28 @@
-[package]
```

Comment:
Line endings 😐 Bumped `version` to 0.0.13, but substantive changes start with the `[dependencies]`.